graphics/farbfeld/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           legacysupport 1.1
+PortGroup           makefile 1.0
+
+name                farbfeld
+version             4
+revision            0
+license             ISC
+
+categories          graphics
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+description         Suckless image format with conversion tools
+long_description    ${description}
+
+homepage            https://tools.suckless.org/${name}/
+platforms           darwin
+
+master_sites        http://dl.suckless.org/farbfeld/
```

Comment:
We should have especially download source URLs by `https://`

Reply:
Yes, `https://dl.suckless.org` works as well.

Reply:
OK, fixed in 8afd6ad8343c16d5ef4696438239f658dc11b9ad, but we should always use HTTPS for everything going forward.  Thanks.

Reply:
Could you fix `https://dl.suckless.org` for dmenu, surf, sic packages as well?

Reply:
Done.